### ISSUE TITLE
Publicize: Custom Message Gets Included with LinkedIn Card

### ISSUE THREAD

**Issue by tmmbecker** [Automattic/wp-calypso#34002]
When sharing a post via Publicize with a Custom Message, the message gets pulled into the LinkedIn card. 

![Screen Shot on 2019-06-14 at 15-08-28](https://user-images.githubusercontent.com/22805865/59532351-6fa83f00-8eb6-11e9-9075-32e06bfb72ee.png)

<img width="549" alt="screenshot" src="https://user-images.githubusercontent.com/22805865/59532394-9797a280-8eb6-11e9-94a7-1c8189414191.png">

#### What I expected

The custom message should display above the image only. 

#### What happened instead

The custom message is included with the post content preview.

2093223-zen

**Comment by tmmbecker** [comment]
Still an issue.

**Comment by cathymcbride** [comment]
related or possibly duplicate issues:  
https://github.com/Automattic/jetpack/issues/11752
https://github.com/Automattic/jetpack/issues/14269
https://github.com/Automattic/wp-calypso/issues/38697
45114-core <-- likely the root issue

**Comment by akmyta** [comment]
Another report was made here: #3741014-zen

**Comment by cathymcbride** [comment]
Looks like this should be fixed in core?

**Comment by tmmbecker** [comment]
@jeherve can you confirm that if this will be fixed in core?

**Comment by jeherve** [comment]
No, I don't think so; this seems to be an issue that's specific to LinkedIn, and not related to the data that's saved for each post. It's most likely something that would need to be updated in the LinkedIn Publicize process that lives on WordPress.com.

**Comment by kaitohm** [comment]
We have more information from 3741014-zd-woothemes

- [Linkedin Post Inspector](https://www.linkedin.com/post-inspector/) shows the correct title.
- The og:title in the code is correct
- Yet, the custom message is showed in Linkedin
- This happens with scheduled posts (at least)
